meson_options.txt

```diff
@@ -16,6 +16,10 @@ option('mkl-threading', type: 'string', value: 'auto',
         description: 'MKL threading method, one of: `seq`, `iomp`, `gomp`, `tbb`')
 option('disable-svml', type: 'boolean', value: false,
         description: 'Disable building against SVML')
+option('disable-highway', type: 'boolean', value: false,
+        description: 'Disables SIMD-optimized operations related to Google Highway(quicksort)')
+option('disable-intel-sort', type: 'boolean', value: false,
+        description: 'Disables SIMD-optimized operations related to Intel-simd-sort(quicksort, argsort, ...)')
```

Comment:
Do we need to list the algorithms here?

Reply:
> Do we need to list the algorithms here?

Not necessary but why not give a small brief to the end user on which operations are related to?

Reply:
Because it's so far from the code, I expect we'll forget to update it 😿 I don't mind either way though, it is *sort* of helpful (pun intended)

Reply:
Good point, done!